source/_integrations/velux.markdown

```diff
@@ -30,6 +30,15 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+## Configuration
+
```

Comment:
```suggestion
```
Otherwise the title is shown twice
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/b625a97a-4f43-4333-91e7-1c610d3274c6)